backup-restore/neo4j/cluster/README.md

```diff
@@ -0,0 +1,48 @@
+# Neo4j cluster Backup & Restore using IBM Storage Fusion
+
+## Prepare for backup and restore 
+
+1. Export the neo4j namespace
+    ```
+        export NEO4J_NAMESPACE='<NEO4J_NAMESPACE>'
```

Comment:
do we really need export command? 
namespace name can be directly passed to  script `./generate_recipe.sh`

Reply:
it is a clean way to do it .